canonical-websites/www.ubuntu.com
# Add security page

## Done

Add new security page

Todo: Find out where this is going to live in the UI

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/security](http://0.0.0.0:8001/security)
- Make sure the page is as per [copy doc](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#)

## Issue / Card

Fixes #2389 Fixes https://github.com/canonical-websites/www.ubuntu.com/pull/2495
Doc: https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#
